assemble/conf/accumulo-site.xml

```diff
@@ -39,6 +39,15 @@
     <name>tserver.memory.maps.native.enabled</name>
     <value>true</value>
   </property>
+  <!-- Used by the PerTableVolumeChooser which is the default general.volume.chooser -->
```

Comment:
I agree. I think we'd want things to work out of the box. I think this is why we previously discussed changing the default back to random instead of per-table.